doc/release/1.6.0-notes.rst

```diff
@@ -0,0 +1,103 @@
+==========================
+SciPy 1.6.0 Release Notes
+==========================
+
+.. note:: Scipy 1.6.0 is not released yet!
+
+.. contents::
+
+SciPy 1.6.0 is the culmination of X months of hard work. It contains
+many new features, numerous bug-fixes, improved test coverage and better
+documentation. There have been a number of deprecations and API changes
+in this release, which are documented below. All users are encouraged to
+upgrade to this release, as there are a large number of bug-fixes and
+optimizations. Before upgrading, we recommend that users check that
+their own code does not use deprecated SciPy functionality (to do so,
+run your code with ``python -Wd`` and check for ``DeprecationWarning`` s).
+Our development attention will now shift to bug-fix releases on the
+1.3.x branch, and on adding new features on the master branch.
+
+This release requires Python 3.6+ and NumPy 1.14.5 or greater.
```

Comment:
I'm not sure if the bump to 3.7 minimum is happening already for the next release, but that can always be adjusted later.